v3.20/ros/jazzy/ros-jazzy-laser-filters/apkbuild_hook.sh

```diff
@@ -1,3 +1,4 @@
 apkbuild_hook() {
   makedepends="${makedepends} ros-jazzy-launch-ros"
+  export MAKEFLAGS="-j1 -l1"
```

Comment:
laser_filter 2.0.8 seems not causing compiler error even if this is not set.
https://github.com/seqsense/aports-ros-experimental/pull/1125
It should be tested on a package/version which originally causes compiler error.

Reply:
The error still occurs on https://github.com/seqsense/aports-ros-experimental/pull/1122 with this patch https://github.com/seqsense/aports-ros-experimental/pull/1126/checks?check_run_id=45603035534

Reply:
https://forums.developer.nvidia.com/t/ros-hector-mapping-c-internal-compiler-error-segmentation-fault-program-cc1plus/199672
This suggested that the failure is due to stack size limit. If this is true, the limit is applied for each g++ process and concurrency doesn't matter.

Reply:
Default stack size limit on the buildfirm was 8192.
I increased the limit to 32768 now.

Reply:
Testing effect of stack size limit at https://github.com/seqsense/aports-ros-experimental/pull/1127

Reply:
At least, the patch in this PR is not needed. So I'll close this PR. I appreciate your help.